plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/ProjectUtils.java

```diff
@@ -126,7 +128,7 @@
 
     // import the projects
     progress.setWorkRemaining(10 * projectFiles.size() + 10);
-    List<IProject> projects = new ArrayList<>(projectFiles.size());
+    Map<String, IProject> projects = new LinkedHashMap<>();
     System.out.printf("Importing %d projects:\n", projectFiles.size());
```

Comment:
should we be using a logger here? or not printing at all?

Reply:
It's testing code, and so shows up on the Travis transcript.
```
Running com.google.cloud.tools.eclipse.appengine.compat.gpe.GpeClassicMigratorTest
Importing 1 projects:
    /home/travis/build/GoogleCloudPlatform/google-cloud-eclipse/plugins/com.google.cloud.tools.eclipse.appengine.compat.test/target/work/data/TaskQueueExamples/.project
Importing 1 projects:
    /home/travis/build/GoogleCloudPlatform/google-cloud-eclipse/plugins/com.google.cloud.tools.eclipse.appengine.compat.test/target/work/data/TaskQueueExamples/.project
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.866 sec - in com.google.cloud.tools.eclipse.appengine.compat.gpe.GpeClassicMigratorTest
```
Logging doesn't add any value here.